src/lib/operators/join_index.cpp

```diff
@@ -182,6 +187,12 @@ void JoinIndex::_perform_join() {
   _write_output_columns(output_columns, _right_in_table, _pos_list_right);
 
   _output_table->append_chunk(output_columns);
+
+  if (performance_data.chunks_scanned_with_index > performance_data.chunks_scanned_without_index) {
```

Comment:
Is this correct? I somehow expected the comparison to be the other way around. Also, should it display the total number of chunks after the `of` then it should be `performance_data.chunks_scanned_without_index + performance_data.chunks_scanned_with_index`?